utils.py

```diff
@@ -7,17 +7,22 @@
 
 def get_map_center():
     """Returns center of the map"""
-    lat = (config.MAP_END[0] + config.MAP_START[0]) / 2
-    lon = (config.MAP_END[1] + config.MAP_START[1]) / 2
-    return lat, lon
+    if isinstance(config.GRID[0], (int, long, float, complex)):
```

Comment:
I'm not a fan of checking everywhere whether config value is an int, or a list. It's better to convert old value to new format in a single place.
Moreover, if GRID can be a list, then GRID[0] may raise an IndexError.